images/varnish-drupal/drupal.vcl

```diff
@@ -147,23 +151,27 @@ sub vcl_recv {
      return (pipe);
   }
 
-  # We only try to cache  GET and HEAD, other things are passed
+  # Large binary files are passed.
+  if (req.url ~ "\.(msi|exe|dmg|zip|tgz|gz|pkg)") {
```

Comment:
Is this going to catch anything that isn't already caught by the `Content-Length` check? I mean, it will catch very small binaries, but does it matter if these are cached?

Reply:
Yeah, this is mainly to prevent the files < 10MB that are pure static binary files. These files are fairly pointless to have in Varnish cache.

Reply:
OK, makes sense. Should you put a `$` at the end of the regex to make it a bit more specific?

Reply:
@seanhamlin ping

Reply:
👍